src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerTest.java

```diff
@@ -49,16 +53,18 @@
   private MockMvc mvc;
 
   @MockBean
-  private ZipService zipService;
+  private HarvestService harvestService;
 
   @MockBean
   private DatasetService datasetService;
 
   @MockBean
   private DatasetReportService datasetReportService;
 
+  private final TestUtils testUtils = new TestUtils();
+
```

Comment:
What about tests for OAI-PMH?

Reply:
Are you planning to add the tests related to the OAI-PMH?

Reply:
Added one test for OAI-PMH with the outcome of expected success.